css/style.css

```diff
@@ -54,6 +62,150 @@ body {
   margin: auto;
 }
 
+main,
+header,
+article,
+aside,
+footer,
+section {
+  padding: 5px;
+  float: left;
```

Comment:
you generally don't want to float all your main containers - just the ones that are side by side. In this case, only float your `main` + `aside`